lib/keila/mailings/builder.ex

```diff
@@ -65,7 +67,11 @@ defmodule Keila.Mailings.Builder do
 
   defp put_template_assigns(assigns, _), do: assigns
 
-  defp process_assigns(value) when is_number(value) or is_binary(value) or is_nil(value) do
+  defp process_assigns(value) when is_number(value) or is_binary(value) do
```

Comment:
These two next functions are not doing the same thing?

Why not group the `when` guards?

Reply:
Good point! I think it evolved organically like that, but they can be/should be grouped.

Reply:
They have been grouped now.